.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,11 @@
 blank_issues_enabled: true
 contact_links:
-  - name: xk6-browser Community Forum
-    url: https://community.k6.io/c/xk6-browser/14
+  - name: Community Forum
```

Comment:
Since we're already on the xk6-browser repo, I believe we should need to repeat the repo name here.